integral_view/templates/update_remote_replication.html

```diff
@@ -100,6 +100,17 @@
 		  </td>
 		</tr>
 		{% endif %}
+
```

Comment:
If they enter this wrongly while creating, do we want to give them a chance to change it? Easy enough to delete and recreate I guess.